src/casp/services/api/services/cell_quota_service.py

```diff
@@ -65,4 +67,25 @@ def get_quota_for_user(self, user: models.User) -> schemas.UserQuota:
             quota_reset_date=quota_reset_date,
             lifetime_quota=user.lifetime_cell_quota,
             remaining_lifetime_quota=remaining_lifetime_quota,
+            quota_increased=user.quota_increased,
         )
+
+    def increase_quota(self, admin_user: models.User, user_for_increase: models.User):
+        """
+        Increase the lifetime quota of the specified user if their lifetime quota has not been
+        increased yet
+
+        :param admin_user: User object of the admin user requesting the quota increase
+        :param user_for_increase: User object to increase quota for
+
+        :return: None
+
+        :raises: AccessDeniedError if admin_user is not an admin
+        """
+        if not admin_user.is_admin:
+            raise exceptions.AccessDeniedError("Access denied.  This is an admin-only endpoint.")
```

Comment:
Seems like you have 2 spaces in the messages description in "...denied.  This..."

Reply:
Oops, that's an old habit